plugins/inputs/ceph/ceph.go

```diff
@@ -15,6 +15,8 @@ import (
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
+// DO NOT REMOVE THE NEXT TWO LINES! This is required to embed the sampleConfig data.
+//
```

Comment:
Please remove those, despite the comment... :-)

Reply:
Removed comment